cmd/schemagen/schemagen.go

```diff
@@ -0,0 +1,155 @@
+package main
+
+import (
+	"bytes"
+	"embed"
+	"flag"
+	"fmt"
+	"github.com/scylladb/gocqlx/v2"
+	"go/format"
+	"html/template"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"unicode"
+
+	"github.com/gocql/gocql"
+	_ "github.com/scylladb/gocqlx/v2/table"
+)
+
+var (
+	cmd          = flag.NewFlagSet(os.Args[0], flag.ExitOnError)
+	flagCluster  = cmd.String("cluster", "127.0.0.1", "a comma-separated list of host:port tuples")
+	flagKeyspace = cmd.String("keyspace", "system", "keyspace to inspect")
+	flagPkgname  = cmd.String("pkgname", "models", "the name you wish to assign to your generated package")
+	flagOutput   = cmd.String("output", "models", "the name of the folder to output to")
+)
+
+var (
+	//go:embed keyspace.tmpl
+	rootFs embed.FS
```

Comment:
You can use `template string` instead.

Reply:
Sure, the go embed accepts 3 things
* embed.FS - if you have many files
* string - if you have a single file and want it as a string
* []byte - if you have a single file